### PR TITLE
67 integrate tailwind css with react email for enhanced email design flexibility

### PR DESCRIPTION
**Pull request by JakeJeongAiden**
**Description** : Add tailwind to react. Subject to change

Screenshot 2024-01-31 at 2 47 51 PM

Image path is set to locally. I'm not sure if emails will have the logo in this case. Doc examples were using 
```
const baseUrl = process.env.VERCEL_URL
  ? `https://${process.env.VERCEL_URL}`
  : ''
```

for path for image.

### CODE REVIEW

**Review by DinDin985 — APPROVED** [review]
lgtm! Nice work on the styling 👍

File: apps/chameleon/components/email-template.tsx [hunk 1]
```diff
@@ -1,18 +1,105 @@
...
+          extend: {
+            colors: {
+              brand: '#007291',
+            },
+          },
+        },
+      }}
+    >
+      <Body className="bg-white">
+        <Container className="flex bg-stone-600 border shadow-md border-white/5 rounded-tl-[80px] rounded-br-[80px] rounded-tr-xl rounded-bl-xl my-auto mx-auto px-2 text-stone-200 text-[14px]">
```
Comment (Balance8) [inline apps/chameleon/components/email-template.tsx:47]: lets not use tailwind arb values. Convert these to as close as possible defaults

File: apps/chameleon/components/email-template.tsx [hunk 2]
```diff
@@ -1,18 +1,105 @@
...
   message,
 }) => (
-  <div>
-    <h1>Welcome, {name}!</h1>
-    <p>{message}</p>
-  </div>
+  <Html>
+    <Head />
+    <Preview>Chameleon, a safe place for cosplayers & photographers.</Preview>
+    <Tailwind
```
Comment (Balance8) [inline apps/chameleon/components/email-template.tsx:35]: I dont think we need to bring in a config here

File: apps/chameleon/components/email-template.tsx [hunk 3]
```diff
@@ -1,18 +1,105 @@
...
+      <Body className="bg-white">
+        <Container className="flex bg-stone-600 border shadow-md border-white/5 rounded-tl-[80px] rounded-br-[80px] rounded-tr-xl rounded-bl-xl my-auto mx-auto px-2 text-stone-200 text-[14px]">
+          <Section>
+            <Heading className="text-rose-100 font-extrabold text-[32px] p-0 my-[30px] mx-0 text-center">
+              Chameleon
+            </Heading>
+            <Img
+              className="mx-auto my-0"
+              //Warning: local image path
+              src={'icons/chameleon-logo-transparent-bg.png'}
```
Comment (Balance8) [inline apps/chameleon/components/email-template.tsx:55]: you need to integrate `src={`${baseUrl}/static/icons/chameleon-logo-transparent-bg.png`}`

File: apps/chameleon/components/email-template.tsx [hunk 4]
```diff
@@ -1,18 +1,105 @@
...
 
 interface EmailTemplateProps {
-  emails: string[]
   name: string
   message: string
-  subject: string
 }
 
+//baseUrl is used to get the logo image <- Warning: local image path currently for demo purposes
+const baseUrl = process.env.VERCEL_URL
```
Comment (Balance8) [inline apps/chameleon/components/email-template.tsx:24]: You need to use the helper function available in https://github.com/Ionia-Devs/chameleon/blob/35fe7b10ce3a71cf4a304a3f54d91d1485ba4234/apps/chameleon/lib/utils.ts#L19-L21

**Review by Balance8 — CHANGES_REQUESTED** [review]
Have you sent the test email to yourself @JakeJeongAiden ? Clean up the code here and if you need help sending it to yourself, read the resend docs or ask @BBruington n

File: apps/chameleon/components/email-template.tsx
Comment (Balance8) [inline apps/chameleon/components/email-template.tsx:1]: Lets go ahead and move these out of the /components folder and move them into their own folder called /email-templates/{{name-of-template.tsx}}

File: apps/chameleon/components/email-template.tsx [hunk 5]
```diff
@@ -1,18 +1,105 @@
...
+  ? `https://${process.env.VERCEL_URL}`
+  : ''
+
 export const EmailTemplate: React.FC<Readonly<EmailTemplateProps>> = ({
```
Comment (Balance8) [inline apps/chameleon/components/email-template.tsx:28]: Lets give it a meaningful name to inform readers what the template does

**Comment by Balance8** [comment]
Your build is failing because you didnt install 

```
pnpm add @react-email/components -E

```

**Comment by BobFromIHT** [comment]
Only note from me is I think the 'Visit Chameleon' button should be centered with the image and header, just flows better on the eyes :)
looks clean though
#

**Review by Balance8 — APPROVED** [review]
👍🏻

**Pull request merged**